src/Core/Controllers/AdminController.php

```diff
@@ -321,6 +321,9 @@ protected function setupView($template)
         $user = Sentinel::getUser();
         $user->setDisabledActions($this->disabledActions);
 
+        $breadcrumbService = $this->app->make('Just\Shapeshifter\Services\BreadcrumbService');
```

Comment:
@johnhout that returns a string. I could do $breadcrumbService = new \Just\Shapeshifter\Services\BreadcrumbService($this->app); but i don't know if that is better / what you mean?